File: begin/table.js


const table = () => {
    let hor = [];
    for (let x = 1; x < 11; x++) {
        hor.push(x);
    }

    console.log(hor.join('  '));

    for (let x = 1; x < 10; x++) {
        let temp = [];
        for (let y = 0; y < 10; y++) {
            if (y == 0) {
                temp.push(hor[x]);
            } else {
                temp.push(temp[0] * hor[y]);
            }
        }
        console.log(temp.join('  '));
    }
}

table();

File: begin/table.py


def table():
    hor = range(1, 11)
    strs = map(lambda x: str(x), hor)

    print('   '.join(strs))

    x = 1
    while x < 10:
        temp = []
        y = 0
        while y < 10:
            if y == 0:
                temp.append(hor[x])
            else:
                temp.append(temp[0] * hor[y])
            y += 1
        
        s = map(lambda m: str(m), temp)
        print("   ".join(s))
        x += 1
    


table()